src/blocks/BoardListBlock.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { extractCompounds } from "utils/compound-utils";
+
+const Empty: React.FC<unknown> = ({ children }) => <p>{children}</p>;
+
+const Item = ({ slug, description, _selected, _onSelected }: ItemProps) => (
+  <>
+    <p onClick={() => _onSelected?.(slug)}>!{slug}</p>
```

Comment:
If you're adding an onClick event you want to make this a `button` for accessibility reasons, even if you end up styling it not to look like a button at all.